tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
is tsconfig.json supposed to be commited to the repo?

Reply:
Yes, this file contains the configuration for compiling to javascript.